src/config/help.ts

```diff
@@ -1,92 +1,60 @@
 // Copyright 2022 @paritytech/polkadot-staking-dashboard authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import { HelpContentRaw } from 'contexts/Help/types';
-
-export const HELP_CONFIG: HelpContentRaw = [
+export const HELP_CONFIG: any = [
   {
     key: 'overview',
     definitions: [
-      // TODO: migrate to just keys and fetch content from i18n
-      // {
-      //   helpKey: 'Supply Staked',
-      //   localeKey: 'overview.definitions.supply_staked',
-      // },
       {
-        title: 'Supply Staked',
-        description: [
-          'The current cumulative supply of {NETWORK_UNIT} being staked globally.',
-          'The percentage of staked {NETWORK_UNIT} is relative to the total supply of {NETWORK_UNIT}.',
-        ],
+        title: 'overview.definitions.supply_staked.title',
```

Comment:
we do not need 2 separate keys for `title` and `description` for these. We can just supply `overview.definitions.supply_staked`, and we know there will be a title and description field within that.

Also, we need another key object for `helpKey` (do a global search to see that it is used to determine the definition content when clicking the help icon).

So this can be simplified to just `key` and `localeKey` for example.